source/computer/commands.rst

```diff
@@ -1,8 +1,33 @@
 常用命令
 ========
 
-Linux 下有成百上千个命令，每个命令都有众多选项。这一节介绍一些常用命令的常用用法。
+Linux/macOS 下有成百上千个命令，每个命令都有众多选项。这一节介绍一些常用命令的常用用法。
+用户可以通过如下方式查询更多命令及其用法。
 
+1.  使用 `tldr <https://tldr.sh/>`__ 命令查询 [**推荐**]
+
+    ``tldr`` 是由社区维护的简化版的命令帮助文档。其以实例的方式展示了命令的
+    常见用法。例如::
+
+        $ tldr mkdir
+
+        mkdir
+
+        Creates a directory.
+
+        - Create a directory in current directory or given path:
+            mkdir directory
+
+        - Create directories recursively (useful for creating nested dirs):
+            mkdir -p path/to/directory
+
+2.  使用 UNIX 提供的 ``man`` 命令查询
+
+    ``man`` 提供了命令的完整语法，因而通常比较冗长。例如::
+
+        $ man mkdir
+
+3.  网页搜索相关命令
```

Comment:
Shall we recommand some websites? e.g.
- google
- https://man.linuxde.net/
- https://tldr.ostera.io/

Reply:
Yes, but i haven't found satisfactory sites.

Reply:
https://man.linuxde.net
this one is good

Reply:
Not as good as I expect, but OK to list it.

Reply:
Done in df6923aa2fb9b2e8a0e3369392d5f244708e4846.